<!DOCTYPE HTML>
<html>

<head>
  <title>Jinjin Tian</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="style/style.css" />
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h1><a href="index.html">Jinjin<span class="logo_colour"> Tian</span></a></h1>
          
          <h2>Ph.D. student, Department of Statistics and Data Science, 
              Carnegie Mellon University</h2>
        </div>
        
      <div id="photo">
        <img src="jinjin.png" width="150" height="150" style = "float:right">
      </div>
      
      </div>
    <div id="menubar">
        
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li><a href="index.html">Home</a></li>
          <li class="selected"><a href="research.html">Research</a></li>
          <li><a href="https://www.dropbox.com/s/wt0q40eohv53lez/CV_Jinjin%20Tian.pdf?dl=0">CV</a></li>
        </ul>
      
  </div>
    </div>
    <div id="site_content">

      <div id="allcontent">
        
        <h2>Papers</h2>
        <ul>
        <p>(<i>Accepted</i>) <b>ADDIS: an adaptive discarding algorithm for online FDR control with conservative nulls.</b> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
           <tt>(NeurIPS)</tt> Neural Information Processing Systems, 2019 &nbsp; 
          <i>J. Tian, A. Ramdas</i> &nbsp;
          <a href="https://arxiv.org/abs/1905.11465">arxiv</a> &nbsp;
          <a href="https://github.com/JINJINT/ADDIS">code</a></p>
        <p>(<i>preprints</i>) <b>Online mutiple testing with FWER control.</b> &nbsp;
        <i>J. Tian, A. Ramdas</i>
        </ul>

        <h2>Ongoing Projects</h2>

        <p><b>Gene co-expression analysis using single cell RNA-seq data</b></p>
        <p>Gene co-expression network (GCN) is of biological interest for the useful prior information it could provide for gene regulatory network. In this project, we aim at building a unified framework to evaluate the existed GCN estimation method, providing inspriration for inference and new methods. We use our findings to address actual scientific question, like the co-expression network pattern of the disease related genes. </p>

        <p><b>Hierarchical online FDR control methods</b></p>
        <p>Current online testing methods for FDR control is poorly fitted in real applications, since large organizations like Google/Microsoft usually runs experiments in lots of different teams and a single error metric is unrealistic. Therefore, we aim at designing new hierarchical online testing algorithms that maintains FDR control both within each group and across groups with some coordination between them.</p> 

        <h2>Projects in undergraduate</h2>
        
        <p><b>Gaussian Graphical Structure Recovery via Knockoffs with Corrupted Data</b></p>
        <p>In this project, we proposed new procedure called CoCoKnockoff to recover the high dimensional gaussian graphical structure with FDR control given data with additive measurement error and unknown population covariance. With theoretical gauratees for FDR control, numerical studies show that CoCoKnockoff enjoy better statistical power compared with directly using the knockoff method. 
        </p>

        <p><b>Recovering the Gaussian Graphical Structures via Knockoffs</b></p>

        <p> Learning the dependence structures in Gaussian graphical models is of fundamental importance in many contemporary applications. In this work, we propose a new procedure based on constructing knockoff variables such that the FDR for graph recovery can be controlled. Numerical studies show that the proposed procedure enjoys better statistical power compared with existing methods.</p>

        <p><b>Brain Task Classification Borrowing Strength from Region Connectivity Network Structure</b></p>

        <p>In this project, we demonstrated that the brain regions connectivity can be a good measure to classify the brain undertaking different tasks: using the brain structure to guide connections design between layers in deep learning model can significantly improve the classification accuracy and prevent overfitting. The results imply the potentials of the structure of network in  classification of realtional data.</p>




        
  </div>
</body>
</html>
